solutions/fscloud/main.tf

```diff
@@ -0,0 +1,36 @@
+########################################################################################################################
+# Resource Group & Service ID
+########################################################################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.4"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
```

Comment:
This is not how to handle resource group in the DA. Look at all our other DAs, they have `resource_group_name` and `existing_resource_group` input variables. They do not have a prefix variable. Please be consistent...

```
module "resource_group" {
  source                       = "terraform-ibm-modules/resource-group/ibm"
  version                      = "1.1.5"
  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
}
```